wolfcrypt/src/rsa.c

```diff
@@ -819,16 +803,15 @@ static int wc_RsaUnPad_ex(byte* pkcsBlock, word32 pkcsBlockLen, byte** out,
 {
     int ret;
 
-    switch (padType)
-    {
+    switch (padType) {
         case WC_RSA_PKCSV15_PAD:
-            WOLFSSL_MSG("wolfSSL Using RSA PKCSV15 padding");
+            //WOLFSSL_MSG("wolfSSL Using RSA PKCSV15 un-padding");
```

Comment:
I have corrected the commenting style. This message spews during benchmark with debug enabled. I personally don't see the value in it being here. I made sure there is a message for the other uncommon pad type for WC_RSA_OAEP_PAD below.